### PR TITLE
Dev v6 fake sf updates

### PR DESCRIPTION
**Pull request by pastika**

### CODE REVIEW

File: python/modules/SoftBDeepAK8SFProducer.py [hunk 1]
```diff
@@ -64,6 +64,18 @@
     }
 }
 
+DeepTop_Fake_SF = {
+    "2016": {
+        (400, 9999) : (0.921, 0.066)
```
Comment (zhenbinwu) [inline python/modules/SoftBDeepAK8SFProducer.py]: I am a bit confused. I am expecting > 1 SF from the plot below. Even inclusive binning, the stat error bar indicating the 400-500 bin still have larger stat. 
image

File: python/modules/SoftBDeepAK8SFProducer.py [hunk 2]
```diff
@@ -107,6 +119,24 @@
     }
 }
 
+DeepW_Fake_SF = {
+    "2016" : {
+        (200, 300) :  (1.161, 0.032),
```
Comment (zhenbinwu) [inline python/modules/SoftBDeepAK8SFProducer.py]: I thought we will go for single value for W mistag as well. Any reason?

Comment (pastika) [reply python/modules/SoftBDeepAK8SFProducer.py]: the values seem to vary more than their statistical uncertainty for W

File: python/modules/SoftBDeepAK8SFProducer.py (hunk 1 above)
Comment (pastika) [reply python/modules/SoftBDeepAK8SFProducer.py]: The first and second bin have similar stats so the final single value is more or less its central average of the 2, why do you expect the average of these values in the plot to be more than 1?

**Comment by zhenbinwu** [comment]
Looks Ok. Merging

**Pull request merged**